Google Scholar.js

```diff
@@ -887,11 +907,43 @@ ItemFactory.prototype.getAttachments = function (doctype) {
 	attachments = [];
 	for (i = 0, ilen = this.attachmentLinks.length; i < ilen; i += 1) {
 		if (!this.attachmentLinks[i]) continue;
-		attachments.push({
-			title:"Google Scholar Linked " + doctype,
-			url:this.attachmentLinks[i],
-			type:"text/html"
-		});
+		if (Zotero.Utilities.setMultiField) {
+			if ("string" === typeof this.attachmentLinks[i]) {
+				// URL (string)
+				if (this.attachmentLinks[i].slice(0, 1) === "/") {
+					this.attachmentLinks[i] = "http://scholar.google.com"+this.attachmentLinks[i];
+				}
+				this.attachmentLinks[i] = this.attachmentLinks[i].replace(/&q=[^&]*/g,"");
+				attachments.push({title:"Google Scholar: " + doctype, type:"text/html",
+								  url:this.attachmentLinks[i]});
```

Comment:
MLZ has mods to domsaver that allow a DOM element to be extracted and rendered in a custom wrapper. Giving that treatment to the text makes it more readable, and makes for consistency across judgments obtained from other sources. In these pages, we're only interested in the public domain text of the judgements, so there isn't a (legal) problem with the refactoring.